### DISCUSSION TITLE
Authelia overhead

### DISCUSSION THREAD

**Discussion by daften** [htpcBeginner/docker-traefik#96] [Support]
If you're talking about the header too large errors, I've found a solution: https://github.com/authelia/authelia/pull/944/files

Also there's this: https://github.com/traefik/traefik/issues/7445

If it's not related, apologies, just wanted to share a potential solution

**Comment by SimpleHomelab** [comment]
Authelia also uses a lot of memory for argon2id hashing. Use SHA512 hashing instead. This is not an issue with this repo. So moving to discussions.
